src/main/java/com/vmware/burp/extension/web/BurpController.java

```diff
@@ -328,6 +328,13 @@ public void sendToSpider(@RequestParam String baseUrl)
       burp.sendToSpider(baseUrl);
    }
 
+   @ApiOperation(value = "Stop Burp Suite", notes = "This will exit Burp Suite.")
+   @RequestMapping(method = GET, value = "/stop")
+   public void exitBurp(){
```

Comment:
Please see my comment in the pull request and if you still think it is definitely useful to have stop functionality, please add all the possible ApiResponse types for the swagger documentation purpose.

Thanks!